### DISCUSSION TITLE
Current Rule Sets for Germany (last updated 17.08.2022)

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-android#194] [Announcements]
### Information about the digital rules 📏 

The following table shows the digital rules for entering Germany (Travel Rules) and the digital domestic rules (Domestic Rules).

**Last updated: 17.08.2022** 🆙 

The rules are continuously updated and updated by the apps regardless of the releases. 

A check for updates takes place within the apps once every 24h when there is an internet connection. A manual check for updates can be done via the settings of the apps.

An overview is published regularly.

### Travel Rules 🗺️ 

<html><body>

Country | Identifier | Type | CertificateType | ValidFrom | ValidTo | Version | SchemaVersion | Description
-- | -- | -- | -- | -- | -- | -- | -- | --
DE | GR-DE-0001 | Acceptance | General | 2022-07-04T09:00:00Z | 2030-06-01T00:00:00Z | 1.0.1 | 1.0.0 | As of June 1, 2022, travelers entering Germany are now  only obligated to register themselves, provide proof of their Covid  status, or quarantine due to the pandemic if they have previously stayed  in an area of variants of concern. No states/regions are currently  classified as areas of variants of concern.
DE | IR-DE-0001 | Invalidation | Vaccination | 2022-05-06T16:00:00Z | 2030-06-01T00:00:00Z | 1.0.3 | 1.0.0 | The certificate must not be issued by an unauthorized party.

### Domestic Rules 🇩🇪 🛍️ 

<html><body>

Country | Identifier | Type | CertificateType | ValidFrom | ValidTo | Version | SchemaVersion | Description
-- | -- | -- | -- | -- | -- | -- | -- | --
DE | IR-DE-0001 | Invalidation | Vaccination | 2022-05-06T16:00:00Z | 2030-06-01T00:00:00Z | 1.0.3 | 1.0.0 | The certificate must not be issued by an unauthorized party.
DE | RR-DE-0001 | Acceptance | Recovery | 2021-07-03T00:00:00Z | 2030-06-01T00:00:00Z | 1.0.0 | 1.0.0 | The positive NAA test result (e.g., PCR) must be older than 28 days.
DE | TR-DE-0005 | Acceptance | Test | 2021-07-03T00:00:00Z | 2030-06-01T00:00:00Z | 1.0.0 | 1.0.0 | The antigen test (e.g., rapid test) must be included in the EU’s "Common list of COVID-19 rapid antigen tests".
DE | RR-DE-0002 | Acceptance | Recovery | 2022-02-24T17:00:00Z | 2030-06-01T00:00:00Z | 1.0.3 | 1.0.0 | The positive NAA test result (e.g., PCR) must be no older than 90 days.
DE | VR-DE-0001 | Acceptance | Vaccination | 2022-03-18T00:00:00Z | 2030-06-01T00:00:00Z | 1.0.2 | 1.0.0 | The vaccination schedule must be complete (e.g., 1/1, 2/2).
DE | TR-DE-0004 | Acceptance | Test | 2021-07-03T00:00:00Z | 2030-06-01T00:00:00Z | 1.0.0 | 1.0.0 | The test result must be negative.
DE | VR-DE-0002 | Acceptance | Vaccination | 2022-03-28T00:00:00Z | 2030-06-01T00:00:00Z | 1.0.3 | 1.0.0 | Only the following vaccines are accepted: AstraZeneca, Biontech, Janssen, Moderna, Novavax.
DE | TR-DE-0001 | Acceptance | Test | 2021-07-03T00:00:00Z | 2030-06-01T00:00:00Z | 1.0.0 | 1.0.0 | This must be an antigen test (e.g., rapid test) or NAA test (e.g., PCR).
DE | TR-DE-0003 | Acceptance | Test | 2021-07-03T00:00:00Z | 2030-06-01T00:00:00Z | 1.0.0 | 1.0.0 | The sample for an NAA test (e.g., PCR) must have been taken no longer than 72 hours ago.
DE | TR-DE-0002 | Acceptance | Test | 2021-07-03T00:00:00Z | 2030-06-01T00:00:00Z | 1.0.0 | 1.0.0 | The sample for an antigen test (e.g., rapid test) must have been taken no longer than 48 hours ago.
DE | VR-DE-5004 | Acceptance | Vaccination | 2022-02-22T08:00:00Z | 2030-06-01T00:00:00Z | 1.0.0 | 1.0.0 | The full vaccination is valid for an unlimited period.
DE | VR-DE-0003 | Acceptance | Vaccination | 2022-02-24T17:00:00Z | 2030-06-01T00:00:00Z | 1.0.6 | 1.0.0 | At least 14 days must have elapsed since completing the  primary course of immunization. A booster shot or vaccination of someone  who recovered from COVID-19 is valid immediately.

</body>
</html>

### Booster Notification Rules 💉 

<html><body>

Country | Identifier | Type | CertificateType | ValidFrom | ValidTo | Version | SchemaVersion | Description
-- | -- | -- | -- | -- | -- | -- | -- | --
DE | BNR-DE-0100 | BoosterNotification | General | 2022-05-04T16:00:00Z | 2030-06-01T00:00:00Z | 1.0.2 | 1.0.0 | The German Standing Committee on Vaccinations (STIKO)  recommends a booster vaccination for everyone age 12 and older who has  not been diagnosed with a SARS-CoV-2 coronavirus infection and who  completed their basic immunization more than three months ago.  The certificates you have saved indicate that these criteria will soon  apply to you and that you have not had a booster vaccination. We  therefore wish to point out this recommendation. (Rule DE-BNR-0100)  This note is based solely on the certificates saved on your smartphone.  The data was processed on your smartphone. No information was sent to  the RKI or to any third parties in this process.
DE | BNR-DE-0110 | BoosterNotification | General | 2022-05-04T16:00:00Z | 2030-06-01T00:00:00Z | 1.0.2 | 1.0.0 | The German Standing Committee on Vaccinations (STIKO)  recommends a booster vaccination for everyone age 12 and older who has  not been diagnosed with a SARS-CoV-2 coronavirus infection and who  completed their basic immunization more than three months ago.  The certificates you have saved indicate that these criteria apply to  you and that you have not had a booster vaccination. We therefore wish  to point out this recommendation. (Rule DE-BNR-0110)  This note is based solely on the certificates saved on your smartphone.  The data was processed on your smartphone. No information was sent to  the RKI or to any third parties in this process.
DE | BNR-DE-0210 | BoosterNotification | General | 2022-05-04T16:00:00Z | 2030-06-01T00:00:00Z | 1.0.2 | 1.0.0 | The German Standing Committee on Vaccinations (STIKO)  recommends that anyone who received their basic immunization through a  single dose of the Janssen vaccine (Johnson & Johnson) and who has  not been diagnosed with a SARS-CoV-2 coronavirus infection get an  additional dose of the vaccine to optimize their basic immunization, if  their Janssen vaccination took place more than four weeks ago.  The certificates you have saved indicate that these criteria apply to  you and that you have not had any additional vaccinations. We therefore  wish to point out this recommendation. (Rule DE-BNR-0210)  This note is based solely on the certificates saved on your smartphone.  The data was processed on your smartphone. No information was sent to  the RKI or to any third parties in this process.
DE | BNR-DE-0200 | BoosterNotification | General | 2022-05-04T16:00:00Z | 2030-06-01T00:00:00Z | 1.0.2 | 1.0.0 | The German Standing Committee on Vaccinations (STIKO)  recommends that anyone who received their basic immunization through a  single dose of the Janssen vaccine (Johnson & Johnson) and who has  not been diagnosed with a SARS-CoV-2 coronavirus infection get an  additional dose of the vaccine to optimize their basic immunization, if  their Janssen vaccination took place more than four weeks ago.  The certificates you have saved indicate that these criteria will soon  apply to you and that you have not had any additional vaccinations. We  therefore wish to point out this recommendation. (Rule DE-BNR-0200)  This note is based solely on the certificates saved on your smartphone.  The data was processed on your smartphone. No information was sent to  the RKI or to any third parties in this process.

</body>
</html>

**Comment by julian-af-19** [comment]
Is the German domestic ruleset (and the signature) publicly available for download? 
I found the Austrain rules here https://dgc-trusttest.qr.gv.at/rules with their signature https://dgc-trusttest.qr.gv.at/rulessig but no equivalent for Germany.
